runelite-client/src/main/java/net/runelite/client/plugins/slayer/SlayerConfig.java

```diff
@@ -223,7 +224,7 @@ default int streak()
 	)
 	default int points()
 	{
-		return -1;
+		return 0;
```

Comment:
Let's not change these defaults--this would not affect users' active configs, and should be handled in the chat command receiver to ensure we're not printing negative values regardless.